src/main/kotlin/com/hibob/academy/employeeFeedback/service/FeedbackService.kt

```diff
@@ -78,4 +78,13 @@ class FeedbackService(private val feedbackDao: FeedbackDao) {
 
         } ?: throw NotFoundException("Feedback not found")
     }
+
+    fun changeToReviewedOrUnreviewed(feedbackId: Int, companyId: Int, employeeId: Int, reviewed: Boolean) {
+        val feedbackInfo = feedbackDao.getFeedbackById(feedbackId, companyId)
```

Comment:
think if you really need this.
This means that you are going to the db twice - once for fetch, once for update. is it necessary?

Reply:
It's unnecessary. I removed it